### ISSUE TITLE
Utilities Folder is not generated: STM32CubeIDE v1.12.0

### ISSUE THREAD

**Issue by EmbeddedTec** [kagovez/I-CUBE-CoreMark#1]
When generating code with STM32CubeIDE v1.12.0, the Utilites folder including the coremark.h etc. is not generated.

In STM32CubeIDE v1.11.2 this workes fine.

Is this a bug in STM32CubeIDE or did the ST packacking format change?

**Comment by Temperature6** [comment]
Yes, I’ve noticed the same thing: when I set the Toolchain to “CMake” in CubeMX the error pops up, but after switching it back to “MDK” the Utilities folder is created and the project builds fine. Looks like the author didn’t account for users who compile with an IDE other than MDK.
